src/vm/eventtrace.cpp

```diff
@@ -6241,6 +6243,79 @@ VOID ETW::LoaderLog::SendModuleEvent(Module *pModule, DWORD dwEventOptions, BOOL
     }
 }
 
+VOID ETW::MethodLog::SendMethodDetailsEvent(MethodDesc *pMethodDesc)
+{
+    CONTRACTL {
+        NOTHROW;
+        GC_NOTRIGGER;
+    } CONTRACTL_END;
+
+    EX_TRY
+    {
+        if(ETW_TRACING_CATEGORY_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_DOTNET_Context, 
+                                        TRACE_LEVEL_INFORMATION, 
```

Comment:
This is information level as its an addon to some events that are sent at non-verbose level. Also, since this event lives in its own keyword, the verbosity is truly controlled at the keyword level, not at the verbosity level for this event.